src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -187,7 +188,7 @@ public override Task OnConnectedAsync(HubConnectionContext connection)
 
             if (connection.User.Identity.IsAuthenticated)
             {
-                var userChannel = _channelNamePrefix + ".user." + connection.User.Identity.Name;
+                var userChannel = _channelNamePrefix + ".user." + connection.User.FindFirst(ClaimTypes.NameIdentifier)?.Value;
```

Comment:
If there isn't a value then we probably shouldn't be subscribing to `_channelNamePrefix + ".user."`